src/components/molecules/Visualizer/Block/Text/index.tsx

```diff
@@ -32,7 +36,7 @@ const TextBlock: React.FC<Props> = ({
 }) => {
   const intl = useIntl();
   const theme = useTheme();
-  const { text, title, markdown, typography } =
+  const { text, title,textBlockPaddingTop,textBlockPaddingBottom,textBlockPaddingLeft,textBlockPaddingRight, markdown, typography } =
```

Comment:
Your linter might not be working quite right. If it was, this would have been corrected on save. Try `yarn lint` and see if this gets fixed

Reply:
it passed the linter

Reply:
oh sorry I pushed that before I install the linter :100: